numpy/core/records.py

```diff
@@ -773,7 +773,48 @@ def fromrecords(recList, dtype=None, shape=None, formats=None, names=None,
 def fromstring(datastring, dtype=None, shape=None, offset=0, formats=None,
                names=None, titles=None, aligned=False, byteorder=None):
     """Create a (read-only) record array from binary data contained in
-    a string"""
+    a string
+
+    Parameters
+    ----------
+    datastring : bytes-like
+        Binary data contained in a string
+    dtype : data-type, optional
+        Valid dtype for all arrays
+    shape : int or tuple of ints, optional
+        Shape of each array.
+    offset : int, optional
+        Position in the buffer to start reading from.
+    formats, names, titles, aligned, byteorder :
+        If `dtype` is ``None``, these arguments are passed to
+        `numpy.format_parser` to construct a dtype. See that function for
+        detailed documentation.
+
+
+    Returns
+    -------
+    np.recarray
+        Read-only record array consisting of data in datastring.
```

Comment:
```suggestion
        Record array view into the data in datastring. This will be readonly if `datastring` is readonly.
```

Reply:
this still is not changed, was that on purpose?